mkdocs/pages/en/devbook/accounts/account-metadata.md

```diff
@@ -0,0 +1,202 @@
+---
+title: Account Metadata
+---
+
+# Adding Metadata to an Account
+
+<Accounts:> can store <metadata:> as key-value pairs.
+
+This tutorial shows how to add metadata to an account, retrieve metadata from the network, and update existing values.
+
+In this example, an account adds a username to itself and then modifies it.
```

Comment:
A `username` looks like a specific property of accounts and might be confusing. How about:

```suggestion
In this example, the string `alice` is attached an account and then changed to `bob`.
```